### PR TITLE
Updated New-CsCallingLineIdentity

### PR DESCRIPTION
**Pull request by JensTrier**
With new parameters ResourceAccount and CompanyName

### CODE REVIEW

File: skype/skype-ps/skype/New-CsCallingLineIdentity.md [hunk 1]
```diff
@@ -192,6 +218,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ResourceAccount
+This parameter specifies the ObjectId of a resource account/online application instance used for Teams Auto Attendant or Call Queue. The outgoing PSTN call will use the phone number defined on the resource account as caller id. For more information about resource accounts please see https://docs.microsoft.com/en-us/microsoftteams/manage-resource-accounts
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsCallingLineIdentity.md:222]:
```suggestion
This parameter specifies the ObjectId of a resource account/online application instance used for Teams Auto Attendant or Call Queue. The outgoing PSTN call will use the phone number defined on the resource account as caller id. For more information about resource accounts please see https://docs.microsoft.com/microsoftteams/manage-resource-accounts
```

**Review by tseward — APPROVED** [review]
Approve pending @get-itips suggestion

**Review by get-itips — COMMENTED** [review]
Checked in 2.2.0-preview and the parameters are still not there, is this going live in the next preview version?

**Comment by JensTrier** [comment]
> Checked in 2.2.0-preview and the parameters are still not there, is this going live in the next preview version?

Yes, they are not in Teams PS Module and will be added in the next preview version. Hopefully by end of Month. They are in SfBO PS

Should I add a note about that?

**Comment by get-itips** [comment]
> > Checked in 2.2.0-preview and the parameters are still not there, is this going live in the next preview version?
> 
> Yes, they are not in Teams PS Module and will be added in the next preview version. Hopefully by end of Month. They are in SfBO PS
> 
> Should I add a note about that?

Maybe if they find the preview note that you are adding we are ok, what do you think @dariomws ?

**Review by dariomws — APPROVED** [review]
@get-itips agree.

If they are available in a preview version released in a few days, I think it's enough.

**Comment by yogkumgit** [comment]
@get-itips

**Pull request merged**